src/schema/comments.ts

```diff
@@ -665,6 +670,19 @@ export const resolvers: IResolvers<any, Context> = {
 
       return markdown.render(trimmed, { mentions });
     },
+    comment: async (
+      _,
+      { id }: { id: string },
+      ctx,
+      info,
+    ): Promise<GQLComment> =>
+      graphorm.queryOneOrFail<GQLComment>(ctx, info, (builder) => ({
+        queryBuilder: builder.queryBuilder.where(
+          `"${builder.alias}"."id" = :id`,
+          { id },
+        ),
+        ...builder,
+      })),
```

Comment:
Should we check that user is author of the comment or has access (for example in private squads)?